provision/core/phpcs/provision.sh

```diff
@@ -16,8 +16,9 @@ function php_codesniff_setup() {
   vvv_info " * Provisioning PHP_CodeSniffer (phpcs), see https://github.com/squizlabs/PHP_CodeSniffer"
 
   noroot mkdir -p /srv/www/phpcs
+  noroot cp -f "/srv/config/phpcs/composer.json" "/srv/www/phpcs/composer.json"
   cd /srv/www/phpcs
-  COMPOSER_BIN_DIR="bin" noroot composer require --update-with-all-dependencies "dealerdirect/phpcodesniffer-composer-installer" "wp-coding-standards/wpcs" "automattic/vipwpcs" "phpcompatibility/php-compatibility" "phpcompatibility/phpcompatibility-paragonie" "phpcompatibility/phpcompatibility-wp" --no-ansi --no-progress
+  COMPOSER_BIN_DIR="bin" noroot composer install --no-ansi --no-progress
```

Comment:
`composer update` would be preferable

Reply:
This one I think I disagree with. The point of `install` over `update` is to ensure that all users are working in identical environments. We should be including a lockfile to _ensure_ every environment is identical.

Reply:
I agree, people would update only when we update the lock file, right?

Reply:
Correct.

Reply:
I disagree, we're creating work for ourselves. We want the latest WPCS, not a specific version from years ago when we last remembered to update the file ( we will not remember to update the file )

Reply:
Also we're coupling our release cycle to that of other projects, VVV users would need a VVV release to get the newer versions without having to manually intervene

Reply:
> ( we will not remember to update the file )

I hate how true this is...